app/models/components/course/experience_points_records_ability_component.rb

```diff
@@ -22,5 +22,6 @@ def allow_read_course_experience_points_records
 
   def allow_read_own_experience_points_records
     can :read, Course::ExperiencePointsRecord, course_user: { user_id: user.id }
+    can :show_user_exp, Course::ExperiencePointsRecord, course_user: { user_id: user.id }
```

Comment:
Do we need both this and the one above?

Reply:
at first, I tried deleting that `can :read` line and as a result, even student cannot see the Exp box in the UserProfileCard..

Now I look again, I immediately know why is that so, and thus that line's not needed anymore